stable/xray/values.yaml

```diff
@@ -100,6 +100,24 @@ xray:
     enabled: false
     # certificateSecretName:
 
+  # Support for open metrics is only available for Artifactory 7.7.x (appVersions) and above.
+  # To enable set `.Values.xray.openMetrics.enabled` to `true`
+  # Refer - https://www.jfrog.com/confluence/display/JFROG/Open+Metrics
+  openMetrics:
+    enabled: true
```

Comment:
I have set it to `true` because on the documentation it says that it should be enabled by default : https://www.jfrog.com/confluence/display/JFROG/Open+Metrics